shows/732 - Stop going to Spam.md

```diff
@@ -0,0 +1,43 @@
+---
+number: 732
+title: "Stop going to Spam: DMARC, SPF and DKIM Explained"
+date: 1708344000810
+url: https://traffic.libsyn.com/syntax/Syntax_-_732.mp3
+youtube_url: https://youtu.be/ERiQc8hoDvo
```

Comment:
@randyrektor Can we starrt using youtube.com urls directly? Its a small thing, but probably helps the SEO google juice linking directly to the page without a redirect